end-user-surveys/README.md

```diff
@@ -0,0 +1,66 @@
+# <img src="https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png" alt="OpenTelemetry Icon" width="45" height=""> OpenTelemetry community surveys
+This document is intended to outline best practices for long running and durable OpenTelemetry community surveys. 
+
+The OpenTelemetry community would like to gather data about end-users including their experiences, preferences, perceptions and opinions about OpenTelemetry. This data can be used to inform priorities within various OpenTelemetry projects. 
+
+# Active Surveys: 
+1: [OpenTelemetry Quarterly Feedback Survey](https://docs.google.com/forms/d/e/1FAIpQLSdKm6oLYRXlZOhEZMVmjoIn4eBToVYNmF6fwpm5GAIipQmPxA/viewform?pli=1)
+   
+- Intent: Gather quantitative data on our end-user community through a simple periodic survey that provides insight with established long running Key Results that we can trend over time. Recruit end-users for future qualitative studies and surveys
+- Survey owners: @sharrmander and @henrikrexed
+- Duration: This survey will be available in perpetuity with some intentional advertising/pushing once a quarter to submit feedback. 
+- Distribution: Survey Results will be summarized on a quarterly basis 
+- [Anonymized raw quarterly feedback survey results](https://docs.google.com/spreadsheets/d/1j2jFe9LLuhJS-NX2xy78ugMo5gDeG-zZeQLOTsgUDTw/edit#gid=0)
+
+# Community survey best practices
+
+### Community survey principles
+- User feedback should be a mix of qualitative and quantitative:
+  - Qualitative to validate/invalidate hypotheses
+  - Quantitative to set a baseline for key results and segment customers
+- All user feedback will be collected anonymously except direct customer interviews
+- Raw data should be transparent to the community/world
+- The end user working group will reach consensus on key milestones, including:
+  - Recurring/perpetual survey design
+  - Survey Hypotheses
+  - Measuring Key Results
+  - Survey Best Practices
+  - Summarizing and reporting results
+
+### Recognized Survey Types: 
```

Comment:
We should review this section. Do we do the second one?

Reply:
Not to my knowledge...

Reply:
Okay I'll remove it